inginious/frontend/templates/admin/admin_users.html

```diff
@@ -0,0 +1,334 @@
+                <button type="submit" name="activate user" data-action="activate" class="btn btn-success">{{ _("Activate") | safe }}</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div id="add_modal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">{{ _("Add user") }} </span></h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            </div>
+            <div class="modal-body">
+                <p id="feedback"></p>
+                <div class="row">
+                    <div class="col-sm-12">
+                        <label class="control-label">{{ _("Username (min. 4 alphanumerical characters) :") }} </label>
+                        <input name="usrname" type="text" class="form-control">
+                    </div>
+                    <div class="col-sm-12">
+                        <label class="control-label">{{ _("Complete name :") }} </label>
+                        <input name="realname" type="text" class="form-control">
+                    </div>
+                    <div class="col-sm-12">
+                        <label class="control-label">{{ _("Email address :") }} </label>
+                        <input name="email" type="text" class="form-control">
+                    </div>
+                    <div class="col-sm-12">
+                        <label class="control-label">{{ _("Password (min. 6 characters) :") }} </label>
+                        <input name="password" type="password" class="form-control">
+                    </div>
+                </div>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">{{_("Cancel")}}</button>
+                <button type="submit" name="Add user" data-action="add_user" class="btn btn-success">{{ _("Add") | safe }}</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div id="binding_modal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">{{ _("Available bindings") }}</span></h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            </div>
+            <div class="modal-body">
+                <div id="binding_content"></div>
+            </div>
+            <div class="modal-footer">
+                <input type="hidden" name="username" id="username">
+                <button type="button" class="btn btn-default" data-dismiss="modal">{{_("Cancel")}}</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
```

Comment:
That's a lot of JS code. This should go in its own js file, and be included in the minified one. Use function names and the onclick attribute instead of setting the event listener at runtime. This will lessen the risk of regression in case an id changes and be more coherent with the remaining of the code.